xformers/ops.py

```diff
@@ -632,6 +676,7 @@ def memory_efficient_attention(
     Supported formats for inputs/outputs:
         [batch, seqlen, num_heads, K]
         [batch, seqlen, K] (Legacy format)
+    Inputs can be non-contiguous
```

Comment:
maybe mention that the only constraint is that the stride of the last dimension should be 1?